js/package.json

```diff
@@ -77,6 +64,7 @@
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.7.6",
+    "@jupyterlab/builder": "^3.0.0",
```

Comment:
This should likely be a higher bound, no?

Reply:
possibly? ill check

Reply:
no obvious reason to narrow this to eg `^3.0.9` has occurred to me. What's your line of thinking on this, @timkpaine?

Reply:
must be something I had to do to get perspective working (but its got lots of custom web pack stuff), so can probably disregard. the cookie cutter does `3.0.0` so that should be fine